### DISCUSSION TITLE
How to install/uninstall? Can it work on my computer? What are these files in "models\checkpoints" folder?

### DISCUSSION THREAD

**Discussion by iIiLviIirea69** [lllyasviel/Fooocus#3259] [Q&A] [answered]
Hello!

Could you please tell me, if I want to uninstall a program without installing it completely, I only need to delete the folder I unzipped from the archive? I don't need to delete anything else?

Could you also please tell me if with my hardware I can use the program on my device or should I go to Colab?
Nvidia GeForce GTX 1050 Ti Dual
Ryzen 5 1600X 3.2Khz 6/12 cores
DDR4 16 GB RAM

Could you please advise in case I leave the program on my computer, are the files in that folder in the screenshot ok? I had problems during the installation. there is a feeling that there is a lot of unnecessary stuff here.
![3](https://github.com/lllyasviel/Fooocus/assets/85225549/09e467f0-aea0-496a-b990-652155a7a001)

**Comment by mashb1t** [comment, marked as answer]
- Foocus is self contained, if you delete D:/DNA Programs/Fooocus, everything is gone
- a 1050 doesn't fulfill the [minimum system requirements](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#minimal-requirement) and you do not have 32GB RAM for `--always-cpu` => your system is not fit to run Fooocus
- you should delete everything, sadly you can't run the tool

I'd propose to use [Colab](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#colab) instead.
